src/main/java/Duke.java

```diff
@@ -1,10 +1,47 @@
+import commands.Command;
+
+import exceptions.InvalidInputException;
+import tasks.TaskList;
+import utils.Parser;
+import utils.Storage;
+import utils.Ui;
+
```

Comment:
Missing headers comments for public classes

Reply:
Header comments will definitely aid in clarity. Will add headers!